stardog/values.yaml

```diff
@@ -23,6 +23,20 @@ stardog:
   users:
   # - name: nameOfTheUser
   #   password: passwordForTheUser
+  disabledUsers:
+  - anonymous
+  databases:
+  # - name: stardog
+  #   options:
+  #   - name: search.enabled
+  #     value: "true"
+  roles:
+  - name: superuser
+    grants:
+    - action: all
+      object: "*:*"
+    users:
+    - admin
```

Comment:
Let's remove this user. The Stardog `admin` user is a special case which has `superuser` rights anyway.